code/datums/shuttles.dm

```diff
@@ -312,6 +319,22 @@
 	credit_cost = 10000
 	danger_level = SHUTTLE_DANGER_SUBPAR
 
+/datum/map_template/shuttle/emergency/funnypod
+	suffix = "funnypod"
+	name = "Comically Large Escape Pod"
+	description = "A bunch of scraped escape pods glued together."
```

Comment:
"scrapped" not scraped. You can scrape a bunch of scrapped escape pods together, but scrapped is the adjective you are looking for here.